auth/realm/jdbc/src/main/java/org/wildfly/security/auth/realm/jdbc/QueryBuilder.java

```diff
@@ -34,7 +37,16 @@
     private List<ColumnMapper> mappers = new ArrayList<>();
     private DataSource dataSource;
 
+    /**
+     *
```

Comment:
Since javadoc was added here, it would also be good to add a description of the constructor, e.g., 

```
/**
     * Construct new instance.
     *
      * @param sql (must not be {@code null})
     * @param parent (must not be {@code null})
     */
```

Reply:
addressed